context/cuvs-bench/get_datasets.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+set -eo pipefail
+
+export CONDA_PREFIX=/opt/conda
```

Comment:
Is this line used? If so, should we activate the appropriate conda environment rather than hardcode this value?